assets/js/src/core/modules/asset/editor/types/folder/tab-manager/tabs/list/toolbar/tools/grid-actions.tsx

```diff
@@ -38,8 +38,8 @@ export const GridActions = (): React.JSX.Element => {
   const { page } = useListPage()
   const { id } = useAsset()
 
-  const { createZipDownload: createZipFolderDownload } = useZipDownload({ type: 'folder' })
-  const { createZipDownload: createZipAssetListDownload } = useZipDownload({ type: 'asset-list' })
+  const { createZipDownload: createZipFolderDownload } = useZipDownload({ elementType: 'asset', type: 'folder' })
```

Comment:
Why do we need to pass the elementType here allthough useZipDownload() is an asset specific action?

Reply:
That is true, I don't know why I didn't notice it in earlier PRs :laughing: . 
I removed the `elementType` in https://github.com/pimcore/studio-ui-bundle/pull/869/commits/d40ea76df5a96c7b361eefa3a260a17dd1ca0abe